src/Windowing/Silk.NET.Windowing.Common/Internals/WindowImplementationBase.cs

```diff
@@ -177,7 +178,22 @@ public WindowBorder WindowBorder
         }
 
         // Other property implementations
-        public bool TransparentFramebuffer => ExtendedOptionsCache.TransparentFramebuffer;
-        public Rectangle<int> BorderSize => IsInitialized ? CoreBorderSize : default;
+        public bool           TransparentFramebuffer => ExtendedOptionsCache.TransparentFramebuffer;
```

Comment:
I I think this formatting got messed up by your IDE

Reply:
Can one of the maintainers fix this? It looks like the original contributor isn't and I wouldn't want their work to go to waste on something small like this :)

Reply:
@Perksey My bad, i forgot to fix it, just committed a fix for the formatting, tell me if something else is wrong